### PR TITLE
Clippy cleanup

### PR DESCRIPTION
**Pull request by thomasschiwietz**

### CODE REVIEW

File: sdl_viewer/src/box_drawer.rs [hunk 1]
```diff
@@ -124,7 +125,7 @@ impl<'a> BoxDrawer<'a> {
     }
 
     // Draws the outline of the box where each vertex is transformed with 'transform'.
-    fn draw_outlines_from_transformation(&self, transform: &Matrix4<f32>, color: &Color) {
+    fn draw_outlines_from_transformation(&self, transform: &Matrix4<f32>, color: Color) {
```
Comment (SirVer) [inline sdl_viewer/src/box_drawer.rs]: Why did you remove the &Color? I think this should remain.

File: sdl_viewer/src/box_drawer.rs [hunk 2]
```diff
@@ -155,11 +156,11 @@ impl<'a> BoxDrawer<'a> {
     // Internally, the box is defined in local coordinates.
     // We the properties of 'cube' to transform the box into world space.
     // Then we use 'world_to_gl' to transform it into clip space.
-    pub fn draw_outlines(&self, cube: &Cube, world_to_gl: &Matrix4<f32>, color: &Color) {
+    pub fn draw_outlines(&self, cube: &Cube, world_to_gl: &Matrix4<f32>, color: Color) {
```
Comment (SirVer) [inline sdl_viewer/src/box_drawer.rs]: same here.

**Comment by SirVer** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

**Comment by thomasschiwietz** [comment]
Fixes #65
